### PR TITLE
DTSRD-1466.Updating schema parameter with hibernate

### PR DESCRIPTION
**Pull request by SabinaHMCTS**
### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by kiran-yenigala-hmcts — DISMISSED** [review]

**Review by kiran-yenigala-hmcts — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/profilesync/repository/ProfileSyncAuditDetailsRepositoryTest.java [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
```
Comment (lukasz-wolski) [inline src/test/java/uk/gov/hmcts/reform/profilesync/repository/ProfileSyncAuditDetailsRepositoryTest.java:5]: Is there a reason we are introducing outdated junit 4 constructs to replace current junit 5 test setup? (lines 5-19)

File: src/test/java/uk/gov/hmcts/reform/profilesync/repository/ProfileSyncAuditRepositoryTest.java [hunk 2]
```diff
@@ -2,18 +2,18 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
```
Comment (lukasz-wolski) [inline src/test/java/uk/gov/hmcts/reform/profilesync/repository/ProfileSyncAuditRepositoryTest.java:5]: Is there a reason we are introducing outdated junit 4 constructs to replace current junit 5 test setup? (lines 5-16)

File: src/test/java/uk/gov/hmcts/reform/profilesync/repository/ProfileSyncConfigRepositoryTest.java [hunk 3]
```diff
@@ -2,16 +2,16 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
```
Comment (lukasz-wolski) [inline src/test/java/uk/gov/hmcts/reform/profilesync/repository/ProfileSyncConfigRepositoryTest.java:5]: Is there a reason we are introducing outdated junit 4 constructs to replace current junit 5 test setup? (lines 5-14)

File: src/test/java/uk/gov/hmcts/reform/profilesync/repository/SyncConfigRepositoryTest.java [hunk 4]
```diff
@@ -2,16 +2,16 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
```
Comment (lukasz-wolski) [inline src/test/java/uk/gov/hmcts/reform/profilesync/repository/SyncConfigRepositoryTest.java:5]: Is there a reason we are introducing outdated junit 4 constructs to replace current junit 5 test setup? (lines 5-14)

**Review by andy-m-b — APPROVED** [review]

**Review by kiran-yenigala-hmcts — APPROVED** [review]

**Pull request merged**